### PR TITLE
Npm release

### PR DESCRIPTION
**Pull request by bobhageman**

### CODE REVIEW

**Review by sanderhollaar — DISMISSED** [review]

File: js/examples/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "irma-backend-examples",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (ivard) [inline js/examples/package.json]: We don't release the examples, so don't necessarily have to update the version here.

File: js/irma-backend/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.5] - 2022-01-24
+### Security
+- Updated depency `eventsource` to v2.0.2
+### Added
+- package-lock.json
```
Comment (ivard) [inline js/irma-backend/CHANGELOG.md]: I don't really understand why `package-lock.json` is added? It was already there if I'm right.

File: js/irma-backend/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,12 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.5] - 2022-01-24
+### Security
+- Updated depency `eventsource` to v2.0.2
```
Comment (ivard) [inline js/irma-backend/CHANGELOG.md]: There is a small spelling mistake here.

File: js/irma-backend/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,12 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.5] - 2022-01-24
```
Comment (ivard) [inline js/irma-backend/CHANGELOG.md]: I think you meant `2023`.

**Review by ivard — CHANGES_REQUESTED** [review]

File: js/irma-backend/CHANGELOG.md (hunk 2 above)
Comment (bobhageman) [reply js/irma-backend/CHANGELOG.md]: You're right, thanks for spotting! Removed it from the changelog.

File: js/irma-backend/CHANGELOG.md [hunk 5]
```diff
@@ -28,6 +32,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Initial release
 
+[0.1.5]: https://github.com/privacybydesign/irma-backend-packages/compare/6746eac...a478de3
```
Comment (ivard) [inline js/irma-backend/CHANGELOG.md:35]: Let's first merge the fix PR and then check whether the git commit is still correct.

**Review by ivard — DISMISSED** [review]

File: js/irma-backend/CHANGELOG.md (hunk 5 above)
Comment (bobhageman) [reply js/irma-backend/CHANGELOG.md:35]: It should and it is:)

**Review by sanderhollaar — APPROVED** [review]

**Review by ivard — APPROVED** [review]

**Pull request merged**